src/main/java/com/liferay/ide/idea/bndtools/BndFileType.java

```diff
@@ -19,7 +19,7 @@
 import com.intellij.openapi.fileTypes.impl.CustomSyntaxTableFileType;
 import com.intellij.openapi.vfs.VirtualFile;
 
-import com.liferay.ide.idea.util.LiferayIcons;
+import icons.LiferayIcons;
```

Comment:
Can you move this class back to another package, it should still be `com.liferay.xxx`

Reply:
Actually I check the source of IntelliJ, we could only use the package and className such as icons.XXXXIcons because the intelliJ already hard-code this rule to make plugin.xml to read it